.travis.yml

```diff
@@ -1,8 +1,17 @@
 language: ruby
 rvm:
-  - 2.0
-  - 2.1
-  - 2.2
-
+- 2.0
+- 2.1
+- 2.2
```

Comment:
I thought that too, but whenever you use the Travis CLI, it changes it to this. So I let it win to stop the continual struggle

On 6 May 2015 03:09, Max Brosnahan <notifications@github.com> wrote:

In .travis.yml<https://github.com/westfieldlabs/apivore/pull/41#discussion_r29690105>:

> @@ -1,8 +1,17 @@
>  language: ruby
>  rvm:
> -  - 2.0
> -  - 2.1
> -  - 2.2
> -
> +- 2.0
> +- 2.1
> +- 2.2


I think these should be indented

—
Reply to this email directly or view it on GitHub<https://github.com/westfieldlabs/apivore/pull/41/files#r29690105>.